providers/manage.rb

```diff
@@ -118,11 +118,23 @@ def chef_solo_search_installed?
         end
 
         if u['ssh_keys']
+          #on suse, i had errors if group didn't exist with the ssh_keys template, so try creating first
+          case node['platform_family']
+          when 'suse'
+            group new_resource.group_name do
+              gid new_resource.group_id
+              action :create
+            end
+          end
```

Comment:
Though I'm not sure if this is a specific-to-suse thing we might want to fix elsewise.